Flat/src/ModulesConfig.cc

```diff
@@ -541,7 +545,7 @@ void PandaAnalyzer::SetDataDir(const char *s)
     bjetregReader->AddVariable("jetHadFrac[hbbjtidx[0]]",&bjetreg_vars[9]);
 
     gSystem->Exec(
-        Form("wget -O %s/trainings/bjet_regression_v0.weights.xml http://t3serv001.mit.edu/~snarayan/pandadata/trainings/bjet_regression_v0.weights.xml",dirPath.Data())
+        Form("wget -q -O %s/trainings/bjet_regression_v0.weights.xml http://t3serv001.mit.edu/~snarayan/pandadata/trainings/bjet_regression_v0.weights.xml",dirPath.Data())
```

Comment:
Was it confirmed that this was causing large log files? I kind of like having non-silent wget so that we can see problems when they araise.